www/templates/control/main-control.html

```diff
@@ -1,8 +1,11 @@
 <ion-view view-title="Profile" ng-class="ionViewBackgroundClass()">
+   <div>
+    <progress ng-if="isDownloading" class=" has-header download" value="{{progress}}" max="100"></progress>
+  </div>
   <ion-content>
     <div class="control-list-item">
       <div class="control-list-text">{{settings.auth.email}}</div>
-      <div ng-click="returnToIntro()" id ="switch-user" class="icon-button-style"><i class="icon ion-log-out"></i></div>
+      <div ng-click="returnToIntro()" id ="switch-user" class="control-icon-button"><i class="icon ion-log-out"></i></div>
```

Comment:
Yeah I wondered if this should have the new class as well, but @krisma's change didn't include it.